manifests/bosh-manifest/deployments/040-collectd.yml

```diff
@@ -0,0 +1,56 @@
+---
+releases:
+- name: collectd
+  sha1: 5e5c196f9c95fe12eb17bba0f40fdaecaddd6424
+  url: https://github.com/alphagov/paas-collectd-boshrelease/releases/download/0.4/collectd-0.4.tgz
```

Comment:
Could you explain in the two commit messages where this URL is used:

- why we're using a fork, so that we know when we can go back to using upstream
- the steps to create and upload the release, so that someone can update it again in future